lib/upipe-modules/upipe_audio_merge.c

```diff
@@ -356,7 +356,7 @@ static void upipe_audio_merge_produce_output(struct upipe *upipe, struct upump *
     uint8_t output_channels = 0;
     UBASE_ERROR(upipe, uref_sound_flow_get_channels(upipe_audio_merge->flow_def, &output_channels));
     if (input_channels != output_channels)
-        upipe_err_va(upipe, "total input channels (%"PRIu64") != output flow def (%d), some will be skipped or blanked!",
+        upipe_verbose_va(upipe, "total input channels (%"PRIu64") != output flow def (%d), some will be skipped or blanked!",
```

Comment:
Don't you think it would be better to print it once in set_flow_def ?

Reply:
Actually the check is wrong to begin with, I think the intention is to make sure the sum(uref_channels) = sum(flow_def_channels)